recipes/android-ci-workflow.md

```diff
@@ -11,30 +11,45 @@ Example Workflow for commits on the main branch of an Android app. The workflow
 5. [Sending a Slack notification with the build status](/recipes/slack-send-build-status.md)
 6. [Filling the cache for upcoming Pull Request builds](/recipes/pull-request-build-caching.md)
 
+## Instructions
+
+Use the yaml below and change the following env var values to match your project settings:
+- `$PROJECT_LOCATION`
+- `$MODULE`
+- `$VARIANT`
+- `$SLACK_WEBHOOK`
+
 ## bitrise.yml
 
 ```yaml
 ---
-format_version: '11'
+format_version: '13'
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 project_type: android
+
+meta:
+  bitrise.io:
+    stack: linux-docker-android-20.04
+    machine_type_id: standard
+
 workflows:
   ci:
     steps:
     - activate-ssh-key@4:
         run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
-    - git-clone@6: {}
-    - cache-pull@2: {}
+    - git-clone@8: {}
+    - restore-gradle-cache@1: {}
     - android-unit-test@1:
         inputs:
-        - project_location: $PROJECT_LOCATION
-        - variant: $VARIANT
+        - project_location: "$PROJECT_LOCATION"
+        - variant: "$VARIANT"
```

Comment:
nit: It's valid YAML without the quotes too

Reply:
Thanks. I did not even notice these added quotes. 

What happened was that I pasted the full yamls into the workflow editor, updated the version numbers via the WF and then copy pasted back the updated yaml into the recipe. It seems like the WF added these but I will remove them as they are just noise.